### DISCUSSION TITLE
Pnpm catalogs

### DISCUSSION THREAD

**Discussion by fuelkoy** [0x80/isolate-package#105] [General]
Hello and thank you for the wonderful package.

I have used this package and recently changed to use pnpm catalogs in pnpm monorepo. Deploying functions produced error 
Build failed with status: FAILURE and message: npm error code EUNSUPPORTEDPROTOCOL
npm error Unsupported URL Type "catalog:": catalog:services

In pnpm-workspace.yaml I'm basicly declaring:
catalogs:
   services:
     ...debendencies...
     
Just asking if it is possible to use pnpm catalogs as they would simplify the package versio consistency in a monorepo. 
Thank you

**Comment by 0x80** [comment]
I am not familiar with the feature, and after a quick read it seems like it wouldn't be too difficult to support.

However, my time to work on isolate-package is very limited, and there are still some essential / fundamental problems to solve, so I don't think I will try to allocate time for this any time soon.

You can of course try to submit a PR if you like.
